### ISSUE TITLE
Hard to read text on confirm email page after email address change

### ISSUE THREAD

**Issue by markmead** [forem/forem#12336]
**Describe the bug**

White text on gray background

**To Reproduce**

- Change your email address
- Click on the link provided in the email

**Expected behavior**

The text should be readable

**Screenshots**

<img width="519" alt="Screenshot 2021-01-19 at 15 18 35" src="https://user-images.githubusercontent.com/50486078/105054290-bafada00-5a69-11eb-9814-275422004a31.png">

**Desktop (please complete the following information):**

- OS, version:
- Browser, version:

**Smartphone (please complete the following information):**

- Device: Macbook Pro
- OS, version: Big Sur, 11
- Browser, version: Chrome, 87

**Additional context**

**Comment by narender2031** [comment]
I would like to work on this.

**Comment by narender2031** [comment]
I investigated the issue, This issue reproduces in `Night Theme`. 
- We are currently using the device error message file to display this error. When we change the theme. text color is almost the same as the background. 

Possible solutions:
- we can implement the custom errors similarly we did for authorization errors with crayons class `crayons-notice--danger registration__error-notice` to the parent div. 

<img width="1438" alt="Screenshot 2021-01-20 at 11 10 57 PM" src="https://user-images.githubusercontent.com/25151772/105213488-c555ca80-5b74-11eb-9a43-d5233f6b73f0.png">


should I create the PR for the solution @rhymes @ludwiczakpawel

**Comment by ludwiczakpawel** [comment]
@narender2031 yea, using `crayons-notice crayons-notice--danger` would be a way to go 👍

Do you want me to assign you to this issue?

**Comment by narender2031** [comment]
yes, Please assign me this issue.

**Comment by ludwiczakpawel** [comment]
Sweet, I'm looking forward for your PR!

**Comment by narender2031** [comment]
This issue is resolved Please close this issue. Thanks
